pgbouncer-mixin/README.md

```diff
@@ -0,0 +1,14 @@
+PgBouncer Mixin
+
+The PgBouncer Mixin is a set of configurable, reusable, and extensible alerts and dashboards based on the metrics exported by the PgBouncer Exporter. The mixin creates recording and alerting rules for Prometheus and suitable dashboard descriptions for Grafana.
```

Comment:
Can we have a sentence describing what PgBouncer is, for those that don't know? Perhaps a link to it, and to the PgBouncer Exporter?